<launch>

    <node pkg="gmapping" type="slam_gmapping" name="slam_gmapping" output="screen">
		<!-- Sensors -->
		<remap from="/scan" to="/base_scan"/>
		<param name="maxRange" value="5.6"/>
		<param name="xmin" value="0.0"/>
		<param name="ymin" value="0.0"/>
		<param name="xmax" value="30.0"/>
		<param name="ymax" value="30.0"/>
	</node>

</launch>